email_handler.py

```diff
@@ -536,6 +539,51 @@ def handle_email_sent_to_ourself(alias, from_addr: str, msg: Message, user):
     )
 
 
+def apply_dmarc_policy(alias: Alias, contact: Contact, msg: Message) -> Optional[str]:
+    dmarc_result = get_dmarc_status(msg)
+    newrelic.agent.record_custom_event(
```

Comment:
Oh didn't know about `newrelic.agent.record_custom_event`, do you know if it allows us to know how many emails have a specific DMARC status in a given period? As the value is a string here, I don't know how NewRelic would handle or whether it'd increase the storage too much (NewRelic pricing is based on storage)

Reply:
Oh I see, thanks for the info.